src/convertToObject.js

```diff
@@ -7,6 +7,20 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
```

Comment:
[CODE STYLE] - The variable name 'result' is too generic. Consider using a more descriptive name like 'styleObject' that clearly indicates the purpose of the variable.